### ISSUE TITLE
The fixes outside the shared-run machinery still read a run with `\s`, and carry a vertical tab or a no-break space off in silence

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#496]
### What minimal example or steps are needed to reproduce the bug?

```css
a { b: c\v !important; }
```

— a vertical tab (spelled here as an escape; the file carries the character) standing between the value's word and the run in front of `!important`. A no-break space goes the same way, and so does either character beside the other runs these families rewrite.

### What minimal configuration is needed to reproduce the bug?

```json
{
	"plugins": [
		"@stylistic/stylelint-plugin"
	],
	"rules": {
		"@stylistic/declaration-bang-space-before": "never"
	}
}
```

### What did you expect to happen?

A vertical tab and a no-break space are words to PostCSS's tokenizer, so the run in front of the bang is the space alone: the fix takes that space away and leaves the character standing, warning about whatever is left to warn about.

### What actually happened?

`--fix` writes `a { b: c!important; }` — the vertical tab is carried off in silence, taken for whitespace by the `\s` the fix reads its run with.

### Notes

The branch of [#494](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/494) gave the shared-run machinery of the declaration rules one whitespace reading, the tokenizer's, and left the other families as they were: `LEADING_WHITESPACE` is read by fourteen rules (the bang, comma, selector-attribute, media and at-rule-name families) and `TRAILING_WHITESPACE` by thirteen more beside two raw-handling utils, and every fix among them that rewrites a run a value or a selector can spell reads it with `\s`. What would close it is the same narrowing, family by family, with a probe of the two characters in every position such a fix rewrites — the tokenizer-true names `LEADING_CSS_WHITESPACE` and `TRAILING_CSS_WHITESPACE` are in `lib/regexps.ts` since that branch, waiting for callers. Measured on either side of it alike.
